app/web/src/features/profile/ProfileTagInput.tsx

```diff
@@ -87,7 +87,7 @@ const useStyles = makeStyles((theme) =>
       borderWidth: 1,
       boxShadow: theme.shadows[3],
       marginTop: theme.spacing(1),
-      zIndex: 1,
+      zIndex: 101,
```

Comment:
Why 101 specifically?

Reply:
ToastUI uses 50 (or was it 30), I thought it was good to give some wiggle room